### ISSUE TITLE
Adaptive Bitrate settings in WebRTC app cause video pixilation

### ISSUE THREAD

**Issue by D3ST1N3D** [ant-media/Ant-Media-Server#4445]
### Short description
Adding adaptive bitrate settings in the WebRTC application is causing the video to pixelate frequently. The attached video shows an example of the aforementioned issue and how it stops happening once the adaptive bitrate settings are removed.

### Environment

* Operating system and version: Linux 
* Ant Media Server version: Enterprise Edition 2.4.3 20220418_2241
* Browser name and version: Google Chrome 

### Steps to reproduce
1. Add the following adaptive bitrate settings on the WebRTC app.
<img width="657" alt="image" src="https://user-images.githubusercontent.com/11080313/193547815-fc9ba899-00ae-4273-9894-3901cd39855d.png">

2. Videos should show frequent pixellating once the settings have been applied and should play properly if the adaptive bitrate settings are removed.

### Expected behavior
Adaptive bitrate settings should allow for a smooth transition for bitrate changes. 

### Actual behavior
With adaptive bitrate settings added for 720p and 1080p, the video will pixelate very frequently.

 

https://user-images.githubusercontent.com/11080313/193544681-7ebd00e0-a5a8-4494-a9b7-1cd8d744f574.mp4

**Comment by mekya** [comment]
I think increasing the bitrate will likely resolve your issue.  
Secondly, you can set the preset value to medium on application settings if you're using soft codec.  Let's schedule this issue and let us send a better insructions.

**Comment by mekya** [comment]
Hi @D3ST1N3D @muratugureminoglu ,

I've made some tests on my end. I agree with you about that. Here is the video that I see pixelating in my tests,
it's 720p and 1500Kbps

https://user-images.githubusercontent.com/3456251/198873649-948002be-4220-4dc3-bae0-1656ff9be988.mp4

It seems that 1500Kbps is not good enough for the test video so I've changed some codec settings but the recommend solution is decresing the resolution 360p and keeping the bitrate same(1500Kbps).   I don't see the pixelating in the following video  

https://user-images.githubusercontent.com/3456251/198873906-31c98f09-89e0-4c06-9c06-186a8d3be9de.mp4

In short, you can just configure the bitrate and resolution according to your quality requirements. If you think, there is a pixelating in the second video or you have questions and would like to discuss, happy to listen you ;)
